Site/ASAB Maestro/Descriptors/mongo.yaml

```diff
@@ -40,5 +40,13 @@ files:
         port: 27017
       replication:
         replSetName: rs0
+
+  # Disable telemetry collection by MongoDB
+  - "conf/mongosh.conf": |
+      enableTelemetry: false
```

Comment:
it is as incosistent as the mongo documentation:
https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/configuration/#std-label-c2c-config
https://www.mongodb.com/docs/mongodb-shell/reference/configure-shell-settings-global/